packages/components/atoms/f-button/test-utils/component-objects/f-button.component.js

```diff
@@ -1,29 +1,26 @@
 const Page = require('@justeat/f-wdio-utils/src/page.object');
-const { buildUrl } = require('@justeat/f-wdio-utils/src/storybook-extensions');
 
 module.exports = class Buttons extends Page {
     constructor () {
         super('atom-folder', 'f-button--button-component');
     }
 
-    load (type='action') {
-        const pageUrl = buildUrl(this.componentType, this.componentName, this.path);
-        this.open(pageUrl);
-
-        if ('action' === type) {
-            this.waitForActionComponent();
-        } else {
-            this.waitForLinkComponent();
-        }
-    }
-
     get actionComponent () { return $('[data-test-id="action-button-component"]'); }
 
     get linkComponent () { return $('[data-test-id="link-button-component"]'); }
 
     open (url) {
         super.open(url);
     }
+
+    load (type = 'action') {
+        if ('action' === type) {
```

Comment:
Super small thing, but this reads a bit strange to me this way round. Reads better as

```suggestion
        if (type === 'action') {
```

Reply:
Sure thing. Just gonna reference [Yoda Conditions](https://en.wikipedia.org/wiki/Yoda_conditions) here, for reference.

Reply:
I've never heard of that before! Thanks for the link 🤣